lib/model/entity/profiel/Profiel.class.php

```diff
@@ -181,16 +184,37 @@ class Profiel extends PersistentEntity implements Agendeerbaar {
 		'muziek' => array(T::String, true),
 		'zingen' => array(T::String, true),
 		'vrienden' => array(T::Text, true),
-		'middelbareSchool' => array(T::String),
+		'middelbareSchool' => array(T::String, true),
 		// novitiaat
-		'novitiaat' => array(T::Text),
+		'novitiaat' => array(T::Text, true),
 		'novitiaatBijz' => array(T::Text, true),
 		'medisch' => array(T::Text, true),
-		'startkamp' => array(T::String),
-		'matrixPlek' => array(T::String),
-		'novietSoort' => array(T::String),
+		'startkamp' => array(T::String, true),
+		'matrixPlek' => array(T::String, true),
+		'novietSoort' => array(T::String, true),
 		'kgb' => array(T::Text, true)
 	);
+	/**
+	 * In $properties_lidstatus kan per property worden aangegeven voor welke lidstatusen deze nodig. Bij wijziging van
+	 * lidstatus wordt een property verwijderd als deze niet langer nodig is.
+	 */
+	public static $properties_lidstatus = [
+		'o_adres' => [LidStatus::Lid, LidStatus::Gastlid, LidStatus::Noviet],
+		'o_postcode' => [LidStatus::Lid, LidStatus::Gastlid, LidStatus::Noviet],
+		'o_woonplaats' => [LidStatus::Lid, LidStatus::Gastlid, LidStatus::Noviet],
+		'o_land' => [LidStatus::Lid, LidStatus::Gastlid, LidStatus::Noviet],
+		'o_telefoon' => [LidStatus::Lid, LidStatus::Gastlid, LidStatus::Noviet],
+		'eetwens' => [LidStatus::Lid, LidStatus::Gastlid, LidStatus::Noviet, LidStatus::Kringel],
```

Comment:
Eetwens alleen voor huidige leden lijkt me prima. Het is misschien wel goed om met het mergen van dit een mail te sturen naar de vab, maalcie en misschien soccie. Om ze hiervan op de hoogte te stellen. Bij oudleden zouden we eventueel een waarschuwing op de maaltijdlijst kunnen zetten. Want je gaat geheid krijgen dat er iemand met allergieën mee wil eten.

Reply:
Inderdaad ja. Het lijkt me voldoende om voor oudleden een waarschuwing in de maaltijdlijst te zetten. Mailen om op de hoogte te stellen van deze wijziging hoeft dan niet eens denk ik